src/splitInteger.test.js

```diff
@@ -4,18 +4,58 @@ const splitInteger = require('./splitInteger');
 
 test(`should split a number into equal parts
   if a value is divisible by a numberOfParts`, () => {
```

Comment:
The test description is incomplete and not properly formatted. It should be a single line string. Consider rephrasing it to something like: `should split a number into equal parts if divisible by numberOfParts`.